tools/install-powershell.ps1

```diff
@@ -2,14 +2,14 @@
 # Licensed under the MIT License.
 <#
 .Synopsis
-    Install PowerShell Core on Windows, Linux or macOS.
+    Install PowerShell on Windows, Linux or macOS.
```

Comment:
It can confuse in the context.

Reply:
I don't think this is confusing and only advanced users would use this